### ISSUE TITLE
Latest version not compiling?

### ISSUE THREAD

**Issue by dmiddlecamp** [Hotaman/OneWireSpark#15]
Hi @Hotaman,

  We're noticing that there seems to be an new recent issue compiling this library in the IDE, did something change recently that would have impacted this?  Anything we can do to help?

Thanks!
David

**Comment by dmiddlecamp** [comment]
oops, nevermind!

**Comment by Hotaman** [comment]
@dmiddlecamp, There was a problem with the Redbear DUO not being recognized which was fixed in v1.4. When I updated the web IDE it acted a little confused for me but I was able to get the latest after a fiddling. 1.3 works with all Particle devices, 1.4 changed the way the platformID is being used, the Core gets it's version, everything else now gets the newer version. Seems to be working fine for everyone at this time.
